packages/tocco-ui/src/EditableValue/typeEditors/StyledDurationEdit.js

```diff
@@ -1,28 +1,44 @@
 import styled from 'styled-components'
 
+import {spaceScale} from '../../utilStyles'
+import {StyledSpan} from '../../Typography'
 import {
-  declareFont,
-  spaceScale
-} from '../../utilStyles'
+  StyledEditableWrapper,
+  StyledInputCss
```

Comment:
Can be written in one line.

Reply:
I prefer readable over compact code. Code Diffs are line-based, hence it is easier to recognize what have changed if each import occupies one line. Single statements per line are easily ordered alphabetically, which can be scanned better.